Automattic/jetpack
# tools: lint-gh-actions.js may as well report yaml syntax errors

#### Changes proposed in this Pull Request:

The yaml parsing library being used produces them, we may as well output
them.

Also, don't try to check the script itself as a yaml file (i.e. we need
`.argv.slice(2)`, not `1`).

#### Jetpack product discussion


None

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Check the output of the "Lint GitHub Actions yaml files" job. It shouldn't report that it's checking the script itself.
* Try introducing a yaml error into one of the workflow files. The script should detect it.
